packages/sakuli-legacy/src/context/sahi/action/handle-action-error.function.ts

```diff
@@ -0,0 +1,46 @@
+import { error, ThenableWebDriver } from "selenium-webdriver";
+import { SahiElementQuery, isSahiElementQuery, sahiQueryToString } from "../sahi-element.interface";
+import {Maybe, ifPresent} from '@sakuli/commons';
+import { AccessorUtil } from "../accessor";
+
+function findQuery(args: any[]): Maybe<SahiElementQuery> {
+    return args.find(isSahiElementQuery);
+}
+
+/**
+ * This function will get an Error and the original function that thrown this error
```

Comment:
The description of what this function does is quite vague.

Maybe explain the rationale of

```ts
await driver.executeScript(`arguments[0].scrollIntoView(false)`, e);
                await original(...args)
                    .then(
                    () => Promise.resolve(),
                    e => {
                        return driver.executeScript(`arguments[0].scrollIntoView(true)`, e).then(() => original(...args))
                    });
```

Also the function will only really try to recover in case of a `MoveTargetOutOfBoundError`.